.env.template

```diff
@@ -1,2 +1,3 @@
 API_URL=https://sandbox.iexapis.com/stable
 API_KEY=abcdefg1234567890
+LOCAL_URL=http://10.72.161.57:3000;
```

Comment:
Please use another name, `LOCAL_URL` does not make sense (we will host our server sometime in the future, and that's not local)